### PR TITLE
Dj rest frame

### PR DESCRIPTION
**Pull request by Baranovsky1986**
add api structure for ratu

### CODE REVIEW

File: ratu/urls.py [hunk 1]
```diff
@@ -3,15 +3,17 @@
 from .views import RuoView
 
 from . import views
```
Comment (criticus) [inline ratu/urls.py:5]: what does this line mean?

File: ratu/views.py [hunk 2]
```diff
@@ -23,3 +53,4 @@ def get(self, request):
         ruo = Ruo.objects.all()[:5]
         serializer = RuoSerializer(ruo, many=True)
         return Response({"ruo": serializer.data})
+
```
Comment (criticus) [inline ratu/views.py:56]: please remove empty line

File: ratu/views.py [hunk 3]
```diff
@@ -1,17 +1,47 @@
...
-
-# Create your views here.
 from django.http import HttpResponse
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from ratu.models.rfop_models import Rfop
 from ratu.models.ruo_models import Ruo
 from ratu.serializers import RfopSerializer
 from ratu.serializers import RuoSerializer
+from ratu.models.ratu_models import Region,District,City,Citydistrict,Street
```
Comment (criticus) [inline ratu/views.py:9]: please add space between models

File: ratu/views.py [hunk 4]
```diff
@@ -1,17 +1,47 @@
...
+from .serializers import RegionSerializer,CitySerializer,CitydistrictSerializer,StreetSerializer,DistrictSerializer
```
Comment (criticus) [inline ratu/views.py:10]: same here

File: data_converter/urls.py [hunk 5]
```diff
@@ -19,5 +19,7 @@
 
 urlpatterns = [
     path('api/', include('ratu.urls')),
-    path('admin/', admin.site.urls),
+    path('admin/', admin.site.urls)
 ]
+
+
```
Comment (Iva-khar) [inline data_converter/urls.py:25]: Delete empty lines

File: ratu/serializers.py [hunk 6]
```diff
@@ -1,5 +1,34 @@
 from rest_framework import serializers
 
+class RegionSerializer(serializers.Serializer):
+    name = serializers.CharField(max_length=120)
```
Comment (Iva-khar) [inline ratu/serializers.py:4]: all 'name' fields in Ratu models have max_length=100

File: ratu/urls.py (hunk 1 above)
Comment (Baranovsky1986) [reply ratu/urls.py:5]: this import was before my changes & use for this line - path('', views.index, name='index'),

File: ratu/views.py (hunk 2 above)
Comment (Baranovsky1986) [reply ratu/views.py]: i did it

File: ratu/views.py (hunk 3 above)
Comment (Baranovsky1986) [reply ratu/views.py]: i did it

File: ratu/views.py (hunk 4 above)
Comment (Baranovsky1986) [reply ratu/views.py]: i did it

File: data_converter/urls.py (hunk 5 above)
Comment (Baranovsky1986) [reply data_converter/urls.py]: i did it

File: ratu/serializers.py (hunk 6 above)
Comment (Baranovsky1986) [reply ratu/serializers.py]: i chenged 120 to 100 in all Serializers

File: ratu/admin.py [hunk 7]
```diff
@@ -1,3 +1,10 @@
 from django.contrib import admin
 from ratu.models.rfop_models import Rfop
-admin.site.register(Rfop)
\ No newline at end of file
+from ratu.models.ratu_models import Region,District,City,Citydistrict,Street
```
Comment (criticus) [inline ratu/admin.py:3]: please add spaces

Comment (Baranovsky1986) [reply ratu/admin.py]: i did it

**Pull request merged**